lib/order.rb

```diff
@@ -0,0 +1,107 @@
+require 'csv'
+
+class Order
+
+  TAX = 0.075
+
+  attr_reader :id
+  attr_accessor :products, :customer, :fulfillment_status
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products
+    @customer = customer
+
+    list_of_statuses = [:pending, :paid, :processing, :shipped, :complete]
+
+    if list_of_statuses.include? fulfillment_status
+      @fulfillment_status = fulfillment_status
+    else
+      raise ArgumentError, "Unknown fulfillment status"
+    end
```

Comment:
I think that we have learned enough versions of conditionals that you could've chosen other forms, but this is actually my favorite -- it's readable! Also it makes total sense that if it's valid, you'll be assigning `@fulfillment_status`, and if invalid, you'd raise an error.